1-exercises/A-accessing-values/exercise1.js

```diff
@@ -5,19 +5,19 @@
 */
```

Comment:
good job, I think you missed mandatory exercises or at least I can not see your solutions , let me know if you have them and want me to review them :)

Reply:
yes i havent complete that time my mandatory